common/static_modifiers/hfe_static_modifiers_global.txt

```diff
@@ -26,4 +26,32 @@ hfe_global_events_3_0 = {
 hfe_global_events_3_1 = {
 	planet_jobs_energy_produces_mult = -0.2
 	country_trade_fee = 0.10
+}
+
+hfe_error_log_spam_removal = {
```

Comment:
What is this?

Reply:
The Error Log recorded several dozen, if not over a hundred, lines telling me how terrible I am to create Economic Category Modifiers (ADD and MULT) without using them in the code. I had to create this unused Modifier in order to remove all of the errors being fired.